src/SFA.DAS.Commitments.Database/Tables/ApprenticeshipUpdate.sql

```diff
@@ -22,7 +22,9 @@
 	[EffectiveFromDate] DATETIME NULL,
 	[EffectiveToDate] DATETIME NULL, 
     [Email] NVARCHAR(200) NULL,
-    [DeliveryModel] TINYINT NULL
+    [DeliveryModel] TINYINT NULL, 
+    [EmploymentPrice] DECIMAL NULL, 
```

Comment:
I think this should be `INT`

Reply:
Yes, I thought that from the draft apprenticeship side, but they have Cost as Decimal. So I followed that pattern. Difficult to know which pattern is correct!

Reply:
I'd go with int as that's the correct type.  The `Apprenticeship` table uses `decimal` for `Cost` and we used `int` for EmploymentPrice ` there.

Reply:
OK, I agree